core/modules/entityreference/tests/entityreference.handlers.test

```diff
@@ -480,12 +480,12 @@ class EntityReferenceHandlersTestCase extends BackdropWebTestCase {
         ),
       ),
     );
-    $this->assertReferencable($field, $referencable_tests, 'Comment handler (comment admin)');
+    $this->assertReferencable($field, $referenceable_tests, 'Comment handler (comment admin)');
```

Comment:
The method name should also be changed.

Reply:
`assertReferencable()` is only used in this file, so I've gone ahead and updated it. However the reason I didn't do so initially was because `Referencable` is used in 3 other function names which are used across 7 different files. So I think that, unfortunately, needs to stay.